sycl/doc/extensions/DataFlowPipes/data_flow_pipes_rev4_proposed.asciidoc

```diff
@@ -170,20 +170,86 @@ The pipe class exposes static member functions for writing a data word to a pipe
 
 Blocking and non-blocking forms of the read and write members are defined, with the form chosen based on overload resolution.
 
+Read/write methods take an ext::oneapi::properties argument, which can contain the latency control properties `latency_anchor_id` and/or `latency_constraint`.
+
+* `sycl::ext::intel::latency_anchor_id<N>`, where `N` is an integer: An ID to associate with the current read/write function call, which can then be referenced by other `latency_constraint` properties elsewhere in the program to define relative latency constaints. ID must be unique within the application, and a diagnostic is required if that condition is not met.
+* `sycl::ext::intel::latency_constraint<A, B, C>`: A tuple of three values which cause the current read/write function call to act as an endpoint of a latency constraint relative to a specified `latency_anchor_id` defined by a different instruction.
+** `A` is an integer: The ID of the target anchor defined on a different instruction through a `latency_anchor_id` property.
+** `B` is an enum value: The type of control from the set {`type::exact`, `type::min`, `type::max`}.
+** `C` is an integer: The relative clock cycle difference between the target anchor and the current function call, that the constraint should infer subject to the type of the control (exact, min, max).
+
 [source,c++,Read write members,linenums]
 ----
+namespace sycl::ext::oneapi {
+
+struct latency_anchor_id {
+  template<int32_t ID>
```

Comment:
Technically, `int32_t` is not required to be supported by all C++ compilers.  Is there a reason we can't use `int` here (and elsewhere below)?

Reply:
I expect that we will change the core SYCL 2020 spec to avoid any requirement that the compiler support `int32_t`.  This means that this pipes extension will probably need to be changed to avoid requiring `int32_t` if we ever plan to upstream it into the core SYCL spec.  If you prefer to wait until we propose this extension as part of the SYCL core spec, that's OK.  We can change it to `int` later.  However, it might be easier to do now?

Reply:
Sure, I can update this PR now.